src/simplewallet/simplewallet.cpp

```diff
@@ -2434,13 +2434,27 @@ bool simple_wallet::set_variable(const std::vector<std::string> &args)
 //----------------------------------------------------------------------------------------------------
 bool simple_wallet::set_log(const std::vector<std::string> &args)
 {
-  if(args.size() > 1)
+  if(args.size() > 1 || args.empty())
```

Comment:
This one removes the ability to see the current log categories.

Reply:
oh yea I forgot about that